x-pack/plugins/ml/public/jobs/components/custom_url_editor/styles/main.less

```diff
@@ -1,4 +1,4 @@
-.ml-custom-url-editor {
+.edit-url-form {
```

Comment:
Suggest to keep `.ml-*` prefixes here, AFAIK the upcoming SASS/BEM-style code will also use prefixes.